fets/model/mlcube/mlcube.yaml

```
@@ -0,0 +1,25 @@
+name: MLCommons FeTS
+description: MLCommons FeTS integration
+authors: 
+ - {name: "MLCommons Best Practices Working Group"}
+ - {name: "Maximilian Zenk (DKFZ)"}
+
+platform:
+  accelerator_count: 0
```

Maybe indicate here how to use GPUs:
```suggestion
  accelerator_count: 0   # 1 if GPU should be used
```

---

I added this indication.